dialogue-core/src/main/java/com/palantir/dialogue/core/UserAgentChannel.java

```diff
@@ -31,6 +31,15 @@
  */
 final class UserAgentChannel implements Channel {
 
+    private static final UserAgent.Agent DIALOGUE_AGENT;
+
+    static {
```

Comment:
slight preference to stick this into a stack method and then initialize `DIALOGUE_AGENT` by calling the method

Reply:
done